examples/cm-repl/cli/src/main.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#[macro_use]
+extern crate serde_json;
+#[macro_use]
+extern crate log;
+
+mod actions;
+
+use clap::{clap_app, crate_version};
+use flexi_logger::{DeferredNow, LogSpecBuilder, Logger};
+use log::Record;
+
+use actions::{add_connection, list_connections, remove_connection};
+
+pub fn log_format(
+    w: &mut dyn std::io::Write,
+    _now: &mut DeferredNow,
+    record: &Record,
+) -> Result<(), std::io::Error> {
+    write!(w, "{}", record.args(),)
+}
+
+fn main() {
+    let matches = clap_app!(cmc =>
+        (version: crate_version!())
+        (about: "Connection manager client (cmc)")
+        (@arg url: +takes_value "Rest api address")
```

Comment:
REST API

Also, can you clarify which REST API is specified here? For example, is this the endpoint for the scabbard REST API?  The Splinter REST API?  Something else?